docs/draft_production_docs/oadp-self-service.md

```diff
@@ -0,0 +1,484 @@
+# OADP Self Service Overview
+
+## Overview
+
+OADP Self Service enables non-administrator users to perform backup and restore operations in their authorized namespaces without requiring cluster-wide administrator privileges. This feature provides secure self-service data protection capabilities while maintaining proper access controls.
+
+### Key Benefits
+
+- Allows users to perform namespace-scoped backup and restore operations
+- Provides users with secure access to backup logs and status information
+- Enables users to create dedicated backup storage locations
+- Maintains cluster administrator control over non-administrator operations through 
```

Comment:
maybe refrain from talking about templates and policies as we are not providing any. Maybe just say "restrictions and enforcements"